src/ext/DifxApp/wixext/WixToolset.DifxApp.wixext.targets

```diff
@@ -7,5 +7,9 @@
        Condition=" Exists('$(MSBuildThisFileDirectory)..\$(WixToolsetExtensionPackageFolder)\$(MSBuildThisFileName).dll') " />
     <UnsupportedWixExtension Include="$(MSBuildThisFileName)"
       Condition=" !Exists('$(MSBuildThisFileDirectory)..\$(WixToolsetExtensionPackageFolder)\$(MSBuildThisFileName).dll') " />
+    <WixLibrary Include="$(MSBuildThisFileDirectory)..\$(WixToolsetExtensionPackageFolder)\difxapp_x86.wixlib"
+       Condition=" Exists('$(MSBuildThisFileDirectory)..\$(WixToolsetExtensionPackageFolder)\difxapp_x86.wixlib') and '$(Platform)' != 'x64' " />
```

Comment:
What if we remove the condition here so the build fails when uses a $(Platform) that difx doesn't support?